src/libs/actions/Policy.ts

```diff
@@ -3987,6 +3987,9 @@ function enablePolicyConnections(policyID: string, enabled: boolean) {
     const parameters: EnablePolicyConnectionsParams = {policyID, enabled};
 
     API.write(WRITE_COMMANDS.ENABLE_POLICY_CONNECTIONS, parameters, onyxData);
+    if (enabled) {
+        navigateWhenEnableFeature(policyID, ROUTES.POLICY_ACCOUNTING.getRoute(policyID));
+    }
```

Comment:
@hayata-suenaga Aren't we not navigating to the feature page for other features? I remember removing this.

Reply:
ah really? can you link a PR that will be removing the navigation? 🙇 😮

Reply:
Here's the PR: https://github.com/Expensify/App/pull/40369 and [this](https://github.com/Expensify/App/issues/39841#issuecomment-2048969008) is the discussion from the design team.

Reply:
seems like your PR was reverted [here](https://github.com/Expensify/App/pull/41176), @allroundexperts 

do you know why that was reverted?

Reply:
Yes. It caused an issue on iOS. I'll raise the PR again with a fix.

Reply:
let me know once you raised the PR so that I can see what change you're making

Reply:
How you getting on here @allroundexperts?

Reply:
@allroundexperts, you can handle this in [your PR](https://github.com/Expensify/App/pull/42026#issuecomment-2108923842). I'm closing this